math/src/traits.rs

```diff
@@ -24,6 +24,15 @@ pub trait ByteConversion {
         Self: std::marker::Sized;
 }
 
+/// Serialize function without options
+/// Used to serialize data to feed it into Fiat Shamir
+/// in some protocols, when ByteConversion is not avaible
+/// For example, when using Curve Points
+pub trait SimpleSerialization {
+    /// Returns the byte representation of the element in big-endian order.
```

Comment:
The `big-endian` may not make sense in certain situations.

Reply:
I forgot to delete that description, it's totally wrong, thanks for pointing that